Description: Fix integration tests unarchive
Author: Lee Garrett <debian@rocketjump.eu>
Forwarded: https://github.com/ansible/ansible/pull/84487
Last-Update: 2025-05-12
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/test/integration/targets/unarchive/tasks/test_different_language_var.yml
+++ b/test/integration/targets/unarchive/tasks/test_different_language_var.yml
@@ -1,10 +1,11 @@
 - name: test non-ascii with different LANGUAGE
   when: ansible_os_family == 'Debian'
   block:
-    - name: install fr language pack
+    - name: install fr language pack on Ubuntu
       apt:
         name: language-pack-fr
         state: present
+      when: ansible_distribution == 'Ubuntu'
 
     - name: create our unarchive destination
       file:
